models/sipnet/R/write_restart.SIPNET.R

```diff
@@ -104,6 +104,11 @@ write_restart.SIPNET <- function(outdir, runid, start.time, stop.time, settings,
     names(analysis.save[[length(analysis.save)]]) <- c("snow")
   }
 
+  if ("LAI" %in% variables) {
+    analysis.save[[length(analysis.save) + 1]] <- new.state$LAI  
+    if (new.state$LAI < 0) analysis.save[[length(analysis.save)]] <- 0
```

Comment:
This also seems like something that should be caught by the SDA analysis rather than in `write.restart`. But @mdietze and @araiho would know better.

Reply:
I don't think so... at least if we plan to use different models in the SDA in the future. In this case the names are specific to SIPNET, so if in the future we start to use ED or something the names may be different and I don't know if the SDA should be responsible for those types of differences.

Reply:
but I'm newer at this so I could be wrong/misreading the problem.

Reply:
@ashiklom can you clarify what you mean by "caught by the SDA?"

Reply:
Yeah, exactly. There's another similar change above that forces woody C to be greater than zero, which I think messes up @araiho's zero-inflated stats. 

If nothing else, punting these kinds of corrections to model specific restart functions means we have to duplicate them for every model, which isn't ideal. But more generally, I think we should inspect why the SDA analysis returns impossible values and fix that. In the meantime, to approve this change, it needs to come with a very loud and clear warning that the SDA returned an illegal value (which we can then quickly convert to an error once the underlying SDA problem is fixed).

Reply:
Just as an FYI, this 0 thing isn't just and Lai issue. this is coded in for pretty much every variable in the script.

Reply:
Also, I just added in a warning in the description of the function about some variables having hard coded values if they produce illegal values.